backend/package.json

```diff
@@ -85,7 +85,6 @@
             "plugin:@typescript-eslint/eslint-recommended",
             "plugin:@typescript-eslint/recommended",
             "plugin:@typescript-eslint/recommended-requiring-type-checking",
-            "prettier/@typescript-eslint",
```

Comment:
This will remove the lint checks.  Some of the packages have a major version increase. Most likely there was a breaking change that we may need to address.